robot/Cumulus/tests/browser/crlps/total_current_year_payments.robot

```diff
@@ -0,0 +1,90 @@
+...             robot/Cumulus/resources/NPSPSettingsPageObject.py
+...             robot/Cumulus/resources/CustomizableRollupsPageObject.py
+...             robot/Cumulus/resources/OpportunityPageObject.py
+...             robot/Cumulus/resources/ObjectMangerPageObject.py
+...             robot/Cumulus/resources/AccountPageObject.py
+...             robot/Cumulus/resources/SchedulePaymentPageObject.py
+...             robot/Cumulus/resources/NPSP.py
+Suite Setup     Run keywords
+...             Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+
+*** Variables ***
+&{contact1_fields}       Email=test@example.com
+&{payment_fields}        Amount=833.34   NumPayments=6   Interval=1
+
+*** Test Cases ***
+
+Total Current Year Payments on Prior Year Pledges
+    [Documentation]       Calculates Total Current Year Payments on Prior Year opportunity Pledges
+
+    [tags]                            unstable                          W-037650                           feature:crlps
+    # Create the required currency field and formula fields
+
+    Validate And Create Required CustomField
+    ...                                                    Object=Account
+    ...                                                    Field_Type=Currency
+    ...                                                    Field_Name=This Year Payments on Past Year Pledges
+
+    Validate And Create Required CustomField
+    ...                                                    Object=Payment
+    ...                                                    Field_Type=Formula
+    ...                                                    Field_Name=Is Opportunity From Prior Year
+    ...                                                    Formula=YEAR( npe01__Opportunity__r.CloseDate ) < YEAR( npe01__Payment_Date__c )
+
+    Enable Customizable Rollups
+
+    # Create a Filter group and CRLP setting after checking prior records do not check if element exists
+
+    Load Page Object                                      Custom                          CustomRollupSettings
+    Navigate To Crlpsettings
+    ${dict}=         Create Dictionary     Name=Old Payments    Description=Includes payments where the Opportunity Close Date year is older than the year of the Payment.
+    ${dict1}=        Create Dictionary     Object=Payment       Field=Paid   Operator=Equals   Value=True
+    ${dict2}=        Create Dictionary     Object=Payment    Field=Is Opportunity From Prior Year     Operator=Equals   Value=True
+    ${filterPools}=  Create List           ${dict1}       ${dict2}
+
+    Create New Filter Setting
+    ...                                                   @{filterPools}
+    ...                                                   &{dict}
+    Enable Customizable Rollups
+    Load Page Object                                       Custom                          CustomRollupSettings
+    Navigate To Crlpsettings
```

Comment:
this as well

Reply:
Same as above, I will talk about in the meeting on Monday with possible demo

Reply:
removed